internal/js/modules/k6/browser/common/frame_session.go

```diff
@@ -629,7 +629,7 @@ func (fs *FrameSession) navigateFrame(frame *Frame, url, referrer string) (strin
 			err = fmt.Errorf("%q: %w", errorText, err)
 		}
 	}
-	return documentID.String(), err
+	return documentID.String(), errorText, err
```

Comment:
```suggestion
	return documentID.String(), err
```

Why do we need to return this error? Can't we throw an error here if we identify that the `errorText` is not empty?

In addition, a DNS error is generally an error on any system, why isn't the case here? Are we sure to not silencing the error somewhere else?